libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/CustomDropdownActions.tsx

```diff
@@ -15,7 +20,7 @@ import {
     useDashboardSelector,
 } from "../../../model";
 
-function useIsConfigButtonVisible(filterDisplayFormRef: ObjRef) {
+function useIsConfigButtonVisible(filterDisplayFormRef: ObjRef, attributes: IAttributeMetadataObject[]) {
```

Comment:
Do we need to provide these attributes via parameter?
What about moving all the added code inside DefaultDashboardAttributeFilter.tsx into this hook?